src/app/utils/date-utils.ts

```diff
@@ -15,5 +15,15 @@ export function getHttpFriendly(date: moment.Moment): string {
 }
 
 export function formatDuration(duration: moment.Duration): string {
-    return moment.utc(moment.duration(duration).asMilliseconds()).format('HH:mm');
+    const durationMs = duration.asMilliseconds();
```

Comment:
why homemade and not a plugin https://github.com/jsmreese/moment-duration-format

Reply:
The plugin would format anything >=24h incorrectly.
For example, 25h would be formatted as 01:00 by the plugin in 'HH:mm'.
I could use 'DD HH:mm' which would extract the number of days, but the format required is 'HH:mm'.

So I'm checking if (duration > 24h) 
- If yes; use the home-made format 
- otherwise; use plugin's formatting